src/components/AttentionBox/AttentionBox.module.scss

```diff
@@ -6,16 +6,15 @@
   position: relative;
```

Comment:
Will check line height with yael before merge  (reminder for double design review)